### PR TITLE
N21-1337 removes deprecated ctl versioning

### PR DESCRIPTION
**Pull request by arnegns**
# Short Description
This PR generates the api again after removed deprecated feature flags and uses for script setup for touched components.

## Links to Ticket and related Pull-Requests
- https://ticketsystem.dbildungscloud.de/browse/N21-1337
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/4927
- https://github.com/hpi-schul-cloud/dof_app_deploy/pull/842

## Changes
Removes feature flags:
- FEATURE_CTL_CONTEXT_CONFIGURATION_ENABLED 
- FEATURE_COMPUTE_TOOL_STATUS_WITHOUT_VERSIONS_ENABLED

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/pages/context-external-tool/ContextExternalToolConfigurator.page.vue [hunk 1]
```diff
@@ -50,195 +48,135 @@ import {
...
-					status: "success",
-				});
-
-				await router.push({ path: contextRoute, query: { tab: "tools" } });
-			}
-		};
-
-		onMounted(async () => {
-			if (props.configId) {
-				// Loading order is important
```
Comment (IgorCapCoder) [inline src/pages/context-external-tool/ContextExternalToolConfigurator.page.vue:206]: is loading order not important anymore? ;)

File: src/serverApi/v3/api.ts [hunk 2]
```diff
@@ -1418,6 +1412,9 @@ export enum CopyApiResponseTypeEnum {
     LernstoreMaterialGroup = 'LERNSTORE_MATERIAL_GROUP',
     LinkElement = 'LINK_ELEMENT',
     LtitoolGroup = 'LTITOOL_GROUP',
+    MediaBoard = 'MEDIA_BOARD',
```
Comment (IgorCapCoder) [inline src/serverApi/v3/api.ts:1415]: why are media board things here?

**Review by IgorCapCoder — DISMISSED** [review]

File: src/serverApi/v3/api.ts (hunk 2 above)
Comment (arnegns) [reply src/serverApi/v3/api.ts:1415]: Ups

File: src/pages/context-external-tool/ContextExternalToolConfigurator.page.vue (hunk 1 above)
Comment (arnegns) [reply src/pages/context-external-tool/ContextExternalToolConfigurator.page.vue:206]: Sure, but the order is the same.. I added the comment again

File: src/serverApi/v3/api.ts (hunk 2 above)
Comment (arnegns) [reply src/serverApi/v3/api.ts:1415]: I generated the api again..

**Review by IgorCapCoder — APPROVED** [review]

**Pull request merged**